report/test/report-generator-test.js

```diff
@@ -70,8 +70,8 @@ describe('ReportGenerator', () => {
 
     it('should inject the report renderer javascript', () => {
       const result = ReportGenerator.generateReportHtml({});
-      assert.ok(result.includes('ReportRenderer'), 'injects the script');
-      assert.ok(result.includes('robustness: \\u003c/script'), 'escapes HTML tags in javascript');
+      assert.ok(result.includes('configSettings.channel||"unknown"'), 'injects the script');
+      assert.ok(result.includes('robustness: <\\/script'), 'escapes HTML tags in javascript');
```

Comment:
I'm not sure this is what we want. 

The previous method preserved the string representations of `</script>` while neutralizing the threat, not altering them. This smells like a terser thing rather than a rollup thing. 

WDYT about keeping our existing sanitization?

Reply:
If these values were inside string literals, the two approaches are equivalent at runtime. This particular example is terser sanitizing text in a comment... and the result is even more readable than what we had it before.

I'm not sure what you're suggesting is possible. How would we disable terser's sanitization?

Reply:
> If these values were inside string literals, the two approaches are equivalent at runtime

Ah OK, that wasn't clear to me from this. Maybe we should have a string literal test instead :)

> I'm not sure what you're suggesting is possible. How would we disable terser's sanitization?

I wasn't trying to suggest disabling, just leaving that line in there. If it's an exact copy for strings though, then SGTM